### DISCUSSION TITLE
Possible to limit unwanted dictionary activation?

### DISCUSSION THREAD

**Discussion by squiresque** [koreader/koreader#8279] [Q&A] [answered]
Because I am most often reading encoded text that has no ordinary words I would appreciate the ability to disable the word selection/ dictionary feature, and have KO reader remember that choice for such books.   Is this possible to do in the current version (running KO Reader (updated to the latest available) on Likebook Alita).  Thank you!

**Comment by Frenzie** [comment, marked as answer]
Settings → Taps & gestures → Long press?

**Reply by squiresque** [reply]
Thank you.  Your suggestion seems to work, and I just don't recall seeing that option before, or I was lost in the gesture manager.
